src/CMakeLists.txt

```diff
@@ -68,6 +68,8 @@ set(conky_sources
     exec.h
     fs.cc
     fs.h
+    gradient.cc
```

Comment:
Since we're including the sources like this, we can remove all references to `BUILD_HSV_GRADIENT` from the CMake files and source. The build option is defined in [ConkyBuildOptions.cmake](https://github.com/brndnmtthws/conky/blob/d2f81cee7ee0bedb4b15b306709216358665e7d0/cmake/ConkyBuildOptions.cmake#L270).